perception_launch/launch/object_segmentation/ground_segmentation/ground_segmentation.launch.py

```diff
@@ -198,14 +289,74 @@ def launch_setup(context, *args, **kwargs):
             (
                 "~/input/pointcloud",
                 "no_ground/oneshot/pointcloud"
-                if bool(pipeline_param["additional_lidars"])
+                if bool(ground_segmentation_param["additional_lidars"])
                 else "no_ground/pointcloud",
             ),
             ("~/output/pointcloud", "/perception/object_segmentation/pointcloud"),
         ],
         extra_arguments=[{"use_intra_process_comms": LaunchConfiguration("use_intra_process")}],
     )
 
+    compare_elevation_map_filter_component = ComposableNode(
+        package="compare_map_segmentation",
+        plugin="compare_map_segmentation::CompareElevationMapFilterComponent",
+        name="compare_elevation_map_filter",
+        remappings=[
+            ("input", "no_ground/oneshot/pointcloud"),
+            ("output", "map_filtered/pointcloud"),
+            ("input/elevation_map", "/map/elevation_map"),
+        ],
+        parameters=[
+            {
+                "map_frame": "map",
+                "map_layer_name": "elevation",
+                "height_diff_thresh": 0.15,
+                "input_frame": "map",
+                "output_frame": "base_link",
+            }
+        ],
+        extra_arguments=[{"use_intra_process_comms": False}],  # can't use this with transient_local
+    )
+
+    downsampling_component = ComposableNode(
+        package="pointcloud_preprocessor",
+        plugin="pointcloud_preprocessor::VoxelGridDownsampleFilterComponent",
+        name="voxel_grid_filter",
+        remappings=[
+            ("input", "map_filtered/pointcloud"),
+            ("output", "voxel_grid_filtered/pointcloud"),
+        ],
+        parameters=[
+            {
+                "input_frame": LaunchConfiguration("base_frame"),
+                "output_frame": LaunchConfiguration("base_frame"),
+                "voxel_size_x": 0.04,
+                "voxel_size_y": 0.04,
+                "voxel_size_z": 0.08,
+            }
+        ],
+        extra_arguments=[{"use_intra_process_comms": LaunchConfiguration("use_intra_process")}],
+    )
+
+    voxel_grid_outlier_filter_component = ComposableNode(
+        package="pointcloud_preprocessor",
+        plugin="pointcloud_preprocessor::VoxelGridOutlierFilterComponent",
+        name="voxel_grid_outlier_filter",
+        remappings=[
+            ("input", "voxel_grid_filtered/pointcloud"),
+            ("output", "/perception/object_segmentation/pointcloud"),
+        ],
+        parameters=[
+            {
+                "voxel_size_x": 0.4,
+                "voxel_size_y": 0.4,
+                "voxel_size_z": 100.0,
+                "voxel_points_threshold": 5,
+            }
+        ],
+        extra_arguments=[{"use_intra_process_comms": LaunchConfiguration("use_intra_process")}],
+    )
```

Comment:
It's not easy to read, so it's better to put these lines in a separete function.

Reply:
fix in https://github.com/tier4/AutowareArchitectureProposal_launcher/pull/129/commits/9ff854ce0da1d0f86d535a0c3563b08b3bdf995c